src/utils/rejection-iterable.ts

```diff
@@ -0,0 +1,18 @@
+import { $$asyncIterator } from 'iterall';
+
+export const createRejectionIterable = (error: Error): AsyncIterator<any> => {
+  return {
+    next() {
+      return this.throw(error);
```

Comment:
A bit confusing to have this call itself - perhaps just `return Promise.reject(error)`?